packages/cli/src/commands/api-generator/watchMode/handleChildProcess.ts

```diff
@@ -0,0 +1,25 @@
+import { type ChildProcessWithoutNullStreams } from "child_process";
+
+export function handleChildProcess(child: ChildProcessWithoutNullStreams): Promise<string> {
+    return new Promise((resolve, reject) => {
+        let scriptOutput = "";
+        let errorOutput = "";
+        child.stdout.setEncoding("utf8");
+        child.stdout.on("data", (data) => {
+            scriptOutput += data;
+            console.log(data);
+        });
+        child.stderr.on("data", (data) => {
+            errorOutput += data;
+            console.error(data);
+        });
+
+        child.on("close", (code) => {
+            if (errorOutput.length > 0) {
```

Comment:
I would use the exit code as source for sucess/failure

Reply:
yes true, changed this.